### PR TITLE
Filling in some test gaps to familiarize myself with fyne and increase coverage

### PR DESCRIPTION
**Pull request by asciifaceman**
Please comment any areas you would like targeted for test coverage and I'll give it a shot, I'm starting off small and inconsequential.

### CODE REVIEW

File: widget/entry_test.go [hunk 1]
```diff
@@ -428,3 +428,34 @@ func TestPasswordEntry_Obfuscation(t *testing.T) {
...
+	entry.Hide()
+
+	entry.Show()
+	assert.Equal(t, entry.Visible(), true)
+
+	for _, child := range Renderer(entry).Objects() {
+		assert.Equal(t, child.Visible(), true)
+	}
+
+	// Reperform the test with text and ensure placeholder is not visible
```
Comment (asciifaceman) [inline widget/entry_test.go]: It would be nice to do this another way, but I can't see how I can determine if a child is the placeholderProvider

Comment (andydotxyz) [reply widget/entry_test.go]: The main provider is `Renderer(entry).(*entryRenderer).text`.
The placeholder provider is `Renderer(entry).(*entryRenderer).text`.

Using WidgetRenderer.Objects() is usually not the best bet as (you have seen) it's just a list of graphical objects.

File: canvas/base_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+func TestBase_Resize(t *testing.T) {
+	b := &baseObject{}
+	targetSize := fyne.NewSize(50, 50)
+	b.Resize(targetSize)
+
+	assert.Equal(t, b.Size(), targetSize)
+}
+
+func TestBase_Move(t *testing.T) {
+	originPosition := fyne.NewPos(0, 0)
```
Comment (paulhovey) [inline canvas/base_test.go]: Might be worth testing with a non-default starting point - if something goes wrong and the code defaults to 0,0 it won't show up here.

**Comment by andydotxyz** [comment]
As a draft PR I cannot merge this - do you want to complete it for merging or add to it later?

I just wanted to check as we head towards the 1.0 release :)

File: canvas/base_test.go (hunk 2 above)
Comment (asciifaceman) [reply canvas/base_test.go]: I agree, I added  a  second test that checks just  this.

**Comment by asciifaceman** [comment]
I figure this  PR should be resolved so I Can move on to working on post 1.0 stuff :)

**Comment by andydotxyz** [comment]
Feel free to close this - opening a new one later is fine and probably easier than rebasing, as it’s been some time.

**Pull request closed without merging**